pkg/resources/prometheusRules.go

```diff
@@ -62,29 +62,6 @@ func (r *AlertReconcilerImpl) ReconcileAlerts(ctx context.Context, client k8scli
 			r.Log.Infof("Operation result", l.Fields{"productName": r.ProductName, "alertName": alert.AlertName, "result": string(or)})
 		}
 	}
-
-	rule := &monitoringv1.PrometheusRule{}
-	err := client.Get(ctx, k8sclient.ObjectKey{
-		Name:      "ksm-monitoring-alerts",
-		Namespace: "redhat-rhoam-middleware-monitoring-operator",
```

Comment:
Since this was hardcoded to use `redhat-rhoam-middleware-monitoring-operator`, would this mean that the associated RHMI alert one is kept ? :thinking:

Reply:
At the moment of doing it, the focus was only on RHOAM. It is hard to recall all details of discussions, but the JIRA description suggests removing those alerts from RHOAM with this line: `Upgrading to an RHOAM version with the alert removed`, but the more I'm thinking about it the less confident I am :thinking: 
If the RHMI alert also were supposed to be removed then there would be a new JIRA ticket created with its own cleanup afterwards. What do you think about this? :thinking:

Reply:
The reason this was only done for RHOAM is the alert was falsely firing during the installs of the operator. As there will be no fresh installs of RHMI there was no need to remove it from the current installs.

Reply:
Cool, if there is no need for the alert to be removed in RHMI, then this comment can be ignored, was just wondering is all :+1:

Reply:
I think we can still verify if this is a consideration for RHMI There is a 2.9 release about to happen so if we need it in until after that release it's just a matter of waiting. And there's no problem to have it sit in 1.8 and remove in 1.9. cc @briangallagher as he's co-ordinating the 2.9 release

Reply:
There does not seem to be an equivalent rhmi rule so I think this is fine to delete. @laurafitzgerald